recipes/openms-meta/conda_build_config.yaml

```
@@ -0,0 +1,5 @@
+python:
+  - 3.9.*  *_cpython
+  - 3.8.*  *_cpython
+python_impl:
+  - cpython
```

```suggestion
  - cpython
c_compiler_version:            # [linux]
  - 9                                      # [linux]
cxx_compiler_version:         # [linux]
  - 9                                      # [linux]
```

You could also try version 11.